### DISCUSSION TITLE
Happy holidays and thank you all 🎄🙏

### DISCUSSION THREAD

**Discussion by muety** [muety/wakapi#721] [Announcements]
It's that time of the year again... 🎄

I had my last workday for 2024 yesterday and am full-on in Christmas mode now. Let me take this chance to get something off my chest again: reflecting on the past couple of years, I just quickly wanted to drop a big **thank you** to all users and supporters of Wakapi! 🙏

It's been more than 5 years since I pushed the first commit (0bd71b77085da62c573b44bb3a39f420840e7157) of Wakapi. Today, we have more than 2.9k GitHub stars, almost 3000 users on wakapi.dev and a great, friendly, little community of contributors. That's something I'm incredibly proud of and thankful for! 😌 I mentioned this before, but maintaining a decently sized open source project and having people who actually find the stuff that I'm doing useful, is something that I've dreamed of for long time.

Of course, I wouldn't mind if I could even benefit from the project financially, but making money with open source is particularly hard. Luckily, this has never been my motivation. Even if there wasn't anyone donating for Wakapi, I'd still keep coding and keep the project alive - as long as it's still fun to me. And so far, it always has been! 🤓

So again, thank you! And happy holidays to all of you who are celebrating :v:.

Btw., new (tiny) release ([v2.12.3](https://github.com/muety/wakapi/releases/tag/2.12.3)) out now :rocket:

**Comment by isabelroses** [comment]
Thank you for all your work. Rest well. And happy holidays. ❤️

**Comment by Grommash9** [comment]
Thanks for your work!

**Comment by muety** [comment]
Btw., wakapi.dev got its 3000th user yesterday! 🙌
